packages/rn-tester/Podfile

```diff
@@ -13,12 +13,12 @@ def pods()
 
   # Enable TurboModule
   prefix_path = "../.."
-  use_react_native!(path:prefix_path)
+  use_react_native!(path: prefix_path, hermes_enabled: true, hermes_path: "#{prefix_path}/../hermes")
```

Comment:
Open question: do we want to have Hermes enabled by default in the RNTester?

CC: @TheSavior

Reply:
That's probably a good choice to make sure we don't break this

Reply:
Why does this override the default at https://github.com/facebook/react-native/blob/941bc0ec195716e6a505a3c3a67f97a87ea9bcdc/scripts/react_native_pods.rb#L9 ?  Is this the path to the rpm, or to the compiler?

Reply:
> Open question: do we want to have Hermes enabled by default in the RNTester?

In macOS’ RNTester we have enabled it by default.

Reply:
> Why does this override the default at
> 
> https://github.com/facebook/react-native/blob/941bc0ec195716e6a505a3c3a67f97a87ea9bcdc/scripts/react_native_pods.rb#L9
> 
> ? Is this the path to the rpm, or to the compiler?

@mhorowitz It’s the path to the `npm` package on disk. The reason it’s overriden is because RNTester in this repo isn’t at the default place that a RN app repo would look like.